spec/controllers/transactions_controller_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe TransactionsController, type: :controller do
+  login_user
```

Comment:
@marderer will be cool to wrap `login_user` into `before` block. check [best practicies](http://www.betterspecs.org/)

Reply:
@AleksSenkou  login_user It is a ```before``` block
[discribe here](https://github.com/marderer/personal-finance/blob/d495d940efae4b9ecedcc22508ed4be7f9e43fa3/spec/support/controller_macros.rb)